diff -up sdlmess-0134/src/tools/tools.mak.nounidasm sdlmess-0134/src/tools/tools.mak
--- sdlmess-0134/src/tools/tools.mak.nounidasm	2009-09-10 15:45:02.000000000 +0200
+++ sdlmess-0134/src/tools/tools.mak	2009-09-17 18:05:12.000000000 +0200
@@ -26,7 +26,6 @@ TOOLS += \
 	romcmp$(EXE) \
 	chdman$(EXE) \
 	jedutil$(EXE) \
-	unidasm$(EXE) \
 	ldresample$(EXE) \
 	ldverify$(EXE) \
 	regrep$(EXE) \
@@ -77,19 +76,6 @@ jedutil$(EXE): $(JEDUTILOBJS) $(LIBUTIL)
 
 
 #-------------------------------------------------
-# unidasm
-#-------------------------------------------------
-
-UNIDASMOBJS = \
-	$(TOOLSOBJ)/unidasm.o \
-
-unidasm$(EXE): $(UNIDASMOBJS) $(LIBUTIL) $(LIBOCORE) $(LIBDASM) $(ZLIB) $(EXPAT)
-	@echo Linking $@...
-	$(LD) $(LDFLAGS) $^ $(LIBS) -o $@
-
-
-
-#-------------------------------------------------
 # ldresample
 #-------------------------------------------------
 
